README.md

```diff
@@ -1,2 +1,18 @@
 # Hello-Microverse
-Microverse project 
+Microverse project
+
+### Built With
+Html<br />
+CSS
+
+### Author
+<a href="https://github.com/kusiLaw">Github</a> \
+<a href="https://twitter.com/kusilaw">Twitter</a> \
+<a href="https://www.linkedin.com/in/lawrence-kusi-55a662104">Linkedin</a>
+
+### Acknowledgments
+Thanks to Microverse
+
+### License
+This Project is MIT licensed
```

Comment:
- You did a great job in your readme structure 👍 . Kindly add the [Getting started section ](https://github.com/microverseinc/readme-template/blob/master/README.md) by telling the user how to have your project running locally on their machine. Example:
## Getting Started
To get a local copy up and running follow these simple example steps.

- How to clone the repo, how to move into the cloned repo,
- Prerequisite: e.g vscode editor, a web browser(chrome, etc).

This will make your readme look more professional 😄

Reply:
I would suggest you install linters again and see what you will get.